service/application/src/main/java/com/solace/maas/ep/event/management/agent/scanManager/rest/DataImportControllerImpl.java

```diff
@@ -59,12 +59,12 @@ public ResponseEntity<String> read(@RequestParam("file") MultipartFile file) {
 
     @Override
     @GetMapping(value = "/export/{scanId}/zip")
+    @SuppressWarnings("PMD.CloseResource")
     public ResponseEntity<InputStreamResource> zip(@PathVariable(value = "scanId") String scanId) {
```

Comment:
Cannot resolve CloseResource violation with try-with-resource. It will cause an early invoke of InputStream.close(), hence causing `ClosedChannelException` on ResponseEntity return.